packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1365,13 +1370,40 @@ class Image extends LeafRenderObjectWidget {
     this.centerSlice
   }) : super(key: key);
 
+  /// The image to display.
   final ui.Image image;
+
+  /// If non-null, require the image to have this width.
+  ///
+  /// If null, the image will pick a size that best preserves its intrinsic
+  /// aspect ratio.
   final double width;
+
+  /// If non-null, require the image to have this height.
+  ///
+  /// If null, the image will pick a size that best preserves its intrinsic
+  /// aspect ratio.
   final double height;
+
+  /// If non-null, apply this color filter to the image before painint.
   final ColorFilter colorFilter;
+
+  /// How to inscribe the image into the place allocated during layout.
   final ImageFit fit;
+
+  /// How to align the image within its bounds.
```

Comment:
This should have more detail since the definition of FractionalOffset probably can't help.